specs/derivation.md

```diff
@@ -907,8 +927,9 @@ To process unsafe payloads, the payload must:
 
 The payload is then processed with a sequence of:
 
-- `engine_newPayloadV2`: process the payload. It does not become canonical yet.
-- `engine_forkchoiceUpdatedV2`: make the payload the canonical unsafe L2 head, and keep the safe/finalized L2 heads.
+- `engine_newPayloadV3`: process the payload. It does not become canonical yet. (Bedrock and Canyon
+payloads MUST use V2)
```

Comment:
If we enforce V2 usage for pre-Ecotone, then we should keep V2 around in the specification, and not overwrite it with V3 specs.
Also, we generally want to keep pre-Ecotone specs around in a compatible way, until Ecotone has been accepted by governance.